pkg/util/tls.go

```diff
@@ -0,0 +1,29 @@
+package util
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"os"
+)
+
+func ReadCertBundle(path string) (*x509.CertPool, error) {
```

Comment:
can you mope this in a new separate package, maybe call it `cert` and then name the function ReadBundle()

We should really move away from dumping everything into util packages as this causes a lot of unwanted side effects via other imports that really should not have to be imported on the remote client, i.e. https://github.com/containers/podman/issues/23818. Fixing that will be quite some work but in the meantime we should not add new things there that are needed by the remote client.

Reply:
Done